client/src/pages/settings/settings.page.js

```diff
@@ -22,17 +22,173 @@ import FormControlLabel from "@mui/material/FormControlLabel";
 import { Link as RouterLink } from "react-router-dom";
 import Button from "@mui/material/Button";
 import Checkbox from "@mui/material/Checkbox";
+import useAuth from "../../hooks/useAuth";
+import bcrypt from "bcryptjs";
 
 const SettingsPage = () => {
+    const { auth, setAuth } = useAuth();
+
     const scrollOffset = -1 * window.innerHeight * 0.1;
 
-    const handleSubmit = (event) => {
+    const handleBasicsSubmit = (event) => {
         event.preventDefault();
         const data = new FormData(event.currentTarget);
-        console.log({
+
+        let json_data = JSON.stringify({
+            firstName: data.get("firstName"),
+            lastName: data.get("lastName"),
             email: data.get("email"),
-            password: data.get("password"),
         });
+        const id = auth._id;
+        console.log(id);
+        try {
+            // Send data to the backend via PUT
+            fetch(`http://localhost:8080/users/${id}`, {
+                method: "PUT",
+                headers: { "Content-Type": "application/json" },
+                body: json_data, // body data type must match "Content-Type" header
+            })
```

Comment:
is this route private? if not, it should be. 

If it is, you will need to use the auth middleware which requires you to send cookies which requires you to add to the fetch options "credentials: include"